testing/sdk_utils.py

```diff
@@ -125,6 +149,31 @@ def is_strict_mode():
 ###
 
 
+def pretty_duration(seconds):
+    """ Returns a user-friendly representation of the provided duration in seconds.
+    For example: 62.8 => "1m2.8s", or 129837.8 => "2d12h4m57.8s"
+    """
+    if seconds is None:
+        return ""
+    ret = ""
+    if seconds >= 86400:
```

Comment:
Sorry, I got prematurely excited. Would it make sense to at least split out the `days` and microseconds then? Not critical though because one ends up making the function more complex than what it should be.